donjo-app/controllers/Covid19.php

```diff
@@ -112,10 +131,31 @@ public function detil_pemudik($id)
 		$data['terdata']['terdata_nama'] = $data['individu']['nik'];
 		$data['terdata']['terdata_info'] = $data['individu']['nama'];
 
+		$data['penduduk'] = $this->penduduk_model->get_penduduk($data['terdata']['id_terdata']);
+		$this->session->set_userdata('nik_lama', $data['penduduk']['nik']);
+
+		$data['dusun'] = $this->wilayah_model->list_dusun();
+		$data['rw'] = $this->wilayah_model->list_rw($data['penduduk']['dusun']);
+		$data['rt'] = $this->wilayah_model->list_rt($data['penduduk']['dusun'], $data['penduduk']['rw']);
+		$data['agama'] = $this->penduduk_model->list_agama();
```

Comment:
Mungkin bisa menggunakan reference_model,  $this->reference_model->list_data($table).

public function list_agama()
	{
		$sql = "SELECT * FROM tweb_penduduk_agama WHERE 1";
		$query = $this->db->query($sql);
		$data = $query->result_array();
		return $data;
	}

public function list_pendidikan()
	{
		$sql = "SELECT * FROM tweb_penduduk_pendidikan WHERE 1";
		$query = $this->db->query($sql);
		$data = $query->result_array();
		return $data;
	}

dll, Bisa ditangani sama fuction ini:

public function list_data($tabel, $kecuali='')
	{
		if (!empty($kecuali))
			$this->db->where("id NOT IN ($kecuali)");
		$data = $this->db->select('*')->order_by('id')->get($tabel)->result_array();
		return $data;
	}

Reply:
Okay. Saya perbaiki ambil dari reference